olp-cpp-sdk-core/src/thread/TaskContinuation.cpp

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2022 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#include <olp/core/thread/TaskContinuation.h>
+
+#include <olp/core/client/CancellationToken.h>
+#include <olp/core/thread/SyncQueue.h>
+#include <olp/core/thread/TaskScheduler.h>
+
+#include <atomic>
+#include <cassert>
+#include <memory>
+#include <stack>
+#include <thread>
+#include <utility>
```

Comment:
Are you sure you need all these includes?

Reply:
removed